Babylon/commands/api/scenarioruns/service/api.py

```diff
@@ -0,0 +1,43 @@
+from typing import Optional
+
+from Babylon.utils.request import oauth_request
+
+
+class ScenarioRunService:
+
+    def __init__(self, state: dict, azure_token: str, spec: Optional[dict] = None):
+        self.state = state
+        self.spec = spec
+        self.azure_token = azure_token
+
+    def logs(self):
+        url = self.state["api"]["url"]
+        organization_id = self.state["api"]["organization_id"]
+        scenariorun_id = self.state["api"]["scenariorun_id"]
```

Comment:
these three lines could be factorized for the whole service :slightly_smiling_face:  and it could be nice to add checks for url and organization_id, just in case :slightly_smiling_face: